test/classes/deinitializers/deinitAndDestruct.good

```diff
@@ -0,0 +1,3 @@
+deinitAndDestruct.chpl:11: error: ambiguous call 'C.deinit()'
+deinitAndDestruct.chpl:2: note: candidates are: C.~C()
+deinitAndDestruct.chpl:6: note:                 C.deinit()
```

Comment:
Sorry, didn't mean that as a "we'll continue discussing it here".  I'm curious what other members of the initializers team think but am not convinced they've seen this question.  And if we're going to continue discussing and adding support for deinitializers anyways, this wouldn't land in your court even if the rest of the initializers team agreed with me